### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by SalemCode8**

### CODE REVIEW

File: .ddev/homeadditions/.gitconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[user]
```
Comment (mabumusa1) [inline .ddev/homeadditions/.gitconfig]: @SalemCode8 please add this file to `.gitignore`, we do not  want your email to be all over the internet

**Review by mabumusa1 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/SiteController.php [hunk 2]
```diff
@@ -67,7 +67,7 @@ public function show(Account $account, Site $site)
      */
     public function create(Account $account)
     {
-        $installs = Install::all();
```
[the remark this answers is not in the document]
Comment (SalemCode8) [reply app/Http/Controllers/SiteController.php:70]: This line is replaced with line that gets only the account installs

File: app/Http/Controllers/UserController.php [hunk 3]
```diff
@@ -27,7 +28,12 @@ public function __construct()
      */
     public function index(Account $account)
     {
-        return view('user.index', ['users' => $account->users]);
+        $canEditOwner = $account->users()
```
[the remark this answers is not in the document]
Comment (SalemCode8) [reply app/Http/Controllers/UserController.php:31]: I saw it, but in this line I wanted to get the number of the users that has a role and the function is working on one user, I didn't  want to make this check using foreach

[the remark this answers is not in the document]
Comment (mabumusa1) [reply app/Http/Controllers/UserController.php:31]: @SalemCode8 okay, if that is case create a new function in the model, it is better for code abstraction

**Review by mabumusa1 — APPROVED** [review]

**Pull request merged**